src/main/resources/application-context.xml

```diff
@@ -40,7 +40,7 @@
 		<value>GuideToPharmacologyLigands</value>
 		<value>GuideToPharmacologyTargets</value>
 		<value>PharmacoDB</value>
-		<value>IUPHAR</value>
+		<!-- <value>IUPHAR</value> --><!-- Renamed to GuideToPharmacologyLigands -->
```

Comment:
Should this remain commented out or does it need to be replaced with the new name?

Reply:
`GuideToPharmacologyLigands` is already used a few lines above. That note was more of an explanation to myself (or anyone else who remembers IUPHAR). I can remove this line, if you would like.

Reply:
Oh okay - I didn't notice the value above.  Maybe it could be moved to just above "GuideToPharmacologyLigands" and the comment could be "IUPHAR renamed to GuideToPharmacologyLigands below" so the comment leads right to the replacement.

Reply:
Maybe I'll just delete it...

Reply:
Okay, if you prefer.  You could add a comment after "GuideToPharmacologyLigands" to say "replaces IUPHAR", if you want to keep that note?

Reply:
Already deleted.

Reply:
Okay, did you want to add the note to the "GuideToPharmacologyLigands" value to have a comment that says "replace IUPHAR" or no?

Reply:
No.

Reply:
Okay